### ISSUE TITLE
🐛 [Bug Report] sometimes,the monster I killed,is still attack me

### ISSUE THREAD

**Issue by nangongzhou** [cmangos/issues#3785]
### Bug Details

I kill some monster, but after I kill him,when his hp is 0,he can still attack me, and I can't choose it for attack

see the picture ,after this mage kill me ,his body can still patrol

not only this one ,i meet some other monsters still can do things like this,I just got this one for picture

### Steps to Reproduce

1.
2.
3.
4.

### Core SHA1 Commit Hash

87e8721f2af08a3376665392ed0707e47b4a86f0

### Database SHA1 Commit Hash

08ee90de86f43f82e03a1646fc3e969551c4d7d3

### Operating System

ubuntu-22.04.5-live-server-amd64

### Client Version

2.4.3 (The Burning Crusade)

**Comment by nangongzhou** [comment]
![Snipaste_2024-11-13_16-17-52](https://github.com/user-attachments/assets/d9ae1da3-e840-439c-a7b7-46d3fde6489f)
picture is here ,this one,after he kill me ,he can still patrol like that

**Comment by cpevors** [comment]
I cannot reproduce at all.

**Comment by al3xc1985** [comment]
is this clean cmangos tbc? or does it have any custom patches like lua? or anything else from somewhere?
or is this any repack of someone u are testing on?

**Comment by nangongzhou** [comment]
> I cannot reproduce at all.

that doesn't happen every time,I play for 10days,It only happened two time,first is like the picture,I kill one Scarlet Archmage,in Eastern Plaguelands,another time is in Dun Morogh ,I kill one ice claw bear.I can say,this bug only happend in field,never happend in dungeon

**Comment by nangongzhou** [comment]
> is this clean cmangos tbc? or does it have any custom patches like lua? or anything else from somewhere? or is this any repack of someone u are testing on?

no,i'm sure,this is clean cmangos,it does't have any custom patches,100% clean

**Comment by YU-Scuti** [comment]
Still happens sometimes

https://www.youtube.com/watch?v=MdyK4vNr578
